### ISSUE TITLE
Document E4S Training in User Docs

### ISSUE THREAD

**Issue by shahzebsiddiqui** [E4S-Project/e4s#90]
@sameershende presented https://www.exascaleproject.org/wp-content/uploads/2022/05/E4S-2022-08-25-Shende_2.pdf as part of the E4S training. I think as part of the user training it would great to have any E4S training be documented in user docs such that one can follow through the training material as pose to reading the slides.

This was reported by attendees that it is hard to follow in slide deck. 

Let's create a separate top-level page perhaps `Training` with a subpage for each training page.

**Comment by sameershende** [comment]
I acknowledge this request. And yes, it would be good to have this. Perhaps, we could use the events page in https://E4S.io for this training information?

**Comment by shahzebsiddiqui** [comment]
Yes I think the announcement and links can be made in e4s.io but the training content should be in e4s.rtfd.io

I can see you can setup slack and other mailing notifications to help announce E4S events to the public.

Usually a combination of home page, slack, Twitter, mailing list should help spread the word along with word of mouth.

**Comment by sameershende** [comment]
Yes, we have updated https://E4S.io with new information.
